pymc3/stats.py

```diff
@@ -265,6 +266,15 @@ def loo(trace, model=None, pointwise=False, reff=1., progressbar=False):
     """
     model = modelcontext(model)
 
+    if reff is None:
+        if trace.nchains == 1:
+            reff = 1.
+        else:
+            eff = pm.effective_n(trace)
+            eff_ave = sum(eff[v] for v in eff) / len(eff)
```

Comment:
This wont work with multivariate RVs. I suggest
`pm.stats.dict2pd(eff,'eff').mean()`

Reply:
Thanks!